spec/support/models.rb

```diff
@@ -10,6 +10,7 @@ class ContactBook < ActiveRecord::Base
   has_many :children, through: :parents
 
   has_many_aggregate :companies, :count, :count, "*"
+  has_many_aggregate :companies, :count_with_dive_limit, :count, "*", max_ids_per_query: 2
```

Comment:
Could we rename this to `count_with_max_ids` or something?

Reply:
sure, that's good point I miss.